src/platform/testing/e2e/cypress/support/form-tester/index.js

```diff
@@ -495,12 +495,12 @@ const testForm = testConfig => {
   } = testConfig;
 
   const skippedTests = Array.isArray(skip) && new Set(skip);
-  const testSuite = skip && !skippedTests ? describe.skip : describe;
+  const testSuite =
+    (skip && !skippedTests) || !Cypress.env('FORMS') ? describe.skip : describe;
```

Comment:
NIT: I think multi-conditional ternaries are slightly difficult to read. 

Do you think there'd be any value in extracting the conditions into a variable? For example:

```
const shouldSkip = (skip && !skippedTests) || !Cypress.env('FORMS');
const testSuite = shouldSkip ? describe.skip : describe;
```

If you do think a change like that would be valuable, then I think it could wait to get piggy-backed on a future PR. I don't think it's worth a full pipeline run for a NIT.

Reply:
With my latest refactor this change is no longer needed, so I'll take it out. But yeah, the ternary is hard to read.